integrations/chroma/tests/test_document_store.py

```diff
@@ -92,6 +92,21 @@ def test_delete_not_empty_nonexisting(self, document_store: ChromaDocumentStore)
 
         assert document_store.filter_documents(filters={"id": doc.id}) == [doc]
 
+    def test_document_store_search(self):
```

Comment:
There's no need to repeat ourselves, we're already in the `TestDocumentStore` class.

Just a small nitpick, not blocking really. Feel free to ignore.

```suggestion
    def test_search(self):
```